backend/python/app/__init__.py

```diff
@@ -37,26 +38,56 @@ def create_app(config_name):
     if type(config_name) is not ScriptInfo:
         app.config.from_object(app_config[config_name])
 
-    app.config["CORS_ORIGINS"] = ["http://localhost:3000"]
+    app.config["CORS_ORIGINS"] = [
+        "http://localhost:3000",
+        "https://uw-blueprint-starter-code.firebaseapp.com",
+        "https://uw-blueprint-starter-code.web.app",
+        re.compile("^https:\/\/uw-blueprint-starter-code--pr.*\.web\.app$"),
+    ]
     app.config["CORS_SUPPORTS_CREDENTIALS"] = True
     CORS(app)
 
-    app.config[
-        "SQLALCHEMY_DATABASE_URI"
-    ] = "postgresql://{username}:{password}@{host}:5432/{db}".format(
-        username=os.getenv("POSTGRES_USER"),
-        password=os.getenv("POSTGRES_PASSWORD"),
-        host=os.getenv("DB_HOST"),
-        db=(
-            os.getenv("POSTGRES_DB_TEST")
-            if app.config["TESTING"]
-            else os.getenv("POSTGRES_DB_DEV")
-        ),
-    )
+    if os.getenv("FLASK_CONFIG") != "production":
+        app.config[
+            "SQLALCHEMY_DATABASE_URI"
+        ] = "postgresql://{username}:{password}@{host}:5432/{db}".format(
+            username=os.getenv("POSTGRES_USER"),
+            password=os.getenv("POSTGRES_PASSWORD"),
+            host=os.getenv("DB_HOST"),
+            db=(
+                os.getenv("POSTGRES_DB_TEST")
+                if app.config["TESTING"]
+                else os.getenv("POSTGRES_DB_DEV")
+            ),
+        )
+    else:
+        app.config["SQLALCHEMY_DATABASE_URI"] = os.getenv("DATABASE_URL")
```

Comment:
Heroku's PostgreSQL add-on sets the connection string in the `DATABASE_URL` env variable